### PR TITLE
Clarify returned structure for fluxes

### PR DESCRIPTION
**Pull request by MTCam**

### CODE REVIEW

File: mirgecom/euler.py [hunk 1]
```diff
@@ -199,6 +199,18 @@ def inviscid_flux(discr, eos, q):
...
+    ----
+    The fluxes are returned as a 2D object array with shape:
+    ``(num_equations, ndim)``.  Each entry in the
+    flux array is a :class:`~meshmode.dof_array.DOFArray`.  This
+    form and shape for the flux data is required by the built-in
+    state data handling mechanism in :mod:`mirgecom.euler`. That
+    mechanism is used by at least
+    :class:`mirgecom.euler.ConservedVars`, and
+    :func:`mirgecom.euler.join_conserved`, and
+    :func:`mirgecom.euler.split_conserved`.
```
Comment (inducer) [inline mirgecom/euler.py:213]:
```suggestion
    .. note::

	      The fluxes are returned as a 2D object array with shape:
	      ``(num_equations, ndim)``.  Each entry in the
	      flux array is a :class:`~meshmode.dof_array.DOFArray`.  This
	      form and shape for the flux data is required by the built-in
	      state data handling mechanism in :mod:`mirgecom.euler`. That
	      mechanism is used by at least
	      :class:`mirgecom.euler.ConservedVars`, and
	      :func:`mirgecom.euler.join_conserved`, and
	      :func:`mirgecom.euler.split_conserved`.
```

**Review by inducer — APPROVED** [review]
Nice, thanks!

**Comment by inducer** [comment]
I take it you didn't like the suggestion I added? (I do submit approving review with comments, and what I mean by that is "this is good to merge subject to these comments".)

**Comment by MTCam** [comment]
> I take it you didn't like the suggestion I added? (I do submit approving review with comments, and what I mean by that is "this is good to merge subject to these comments".)

Oh what?!  I didn't even see that!  My apologies!

File: mirgecom/euler.py (hunk 1 above)
Comment (matthiasdiener) [reply mirgecom/euler.py:213]: The indentation should not be tabs here right?

Comment (MTCam) [reply mirgecom/euler.py:213]: Ugh, not sure.  I failed to see this suggestion and then merged without resolving.  Is there any way to recover gracefully?

Comment (MTCam) [reply mirgecom/euler.py:213]: #223 🤦

Comment (inducer) [reply mirgecom/euler.py:213]: > The indentation should not be tabs here right?

Yup, very true.

Comment (MTCam) [reply mirgecom/euler.py:213]: I fixed the formatting in #223

Comment (inducer) [reply mirgecom/euler.py:213]: #224

**Comment by inducer** [comment]
No big deal. Thanks for following up.

**Pull request merged**